### DISCUSSION TITLE
Syncing between local and production server (via git) causes two users to be created

### DISCUSSION THREAD

**Discussion by vhbelvadi** [statamic/cms#7738] [Q&A]
I'm on the free tier of Statamic which means I only have access to one user and I've created them using `php please make:user`.

However, after syncing my statamic folder to my production server I noticed that the same password didn't work. Apparently there were two user accounts after the git sync?

Either way, the question now: *is there any way to sync user accounts between local and production?* Alternately, what's a method to avoid this seeming duplication.

**Comment by edalzell** [comment]
That's a weird git issue, how are you syncing?

**Reply by vhbelvadi** [reply]
I installed Static locally and ran it with Larval Valet for dev stuff. Then I synced it to Github from where I pull to my VPS via Git.

**Comment by northlandz786** [comment]
When you[ synchronize your codebase between your local development environment](https://wbm.com.pk/category/insect-killer) and your production server via Git, it should not create any users by default. However, there could be a few scenarios where users are created during the synchronization process.

One possible scenario could be that you have added a new user to your local development environment, and this user is included in the Git repository. When you push the changes to the production server, the new user will be added to the production server's user database.

Another possible scenario could be that your application's code creates a new user automatically when it is deployed to the production server. This could happen if your application has a feature that allows users to sign up and create accounts.

**Comment by joshuablum** [comment]
Hey @vhbelvadi,

Did you run `php please make:user` on both your local and production environments? So essentially twice?
If so, you created two users with different IDs.

You can do that and `gitignore` the `users` directory or run it locally _once_ and push that user to your server for example.

And there is also [the Gitamic addon](https://statamic.com/addons/simonhamp/gitamic) for $19 that gives you the Git features without the need to buy a Pro license.
